src/server/enrollment-system.js

```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
           addressTypeCode: 'P',  // TODO(awong): this code is from VHA Standard Data Service (ADRDEV01) Address Type List P==Permanent. Determine if we need it.
         },
         emails: {
-          email: veteran.email,
+          email: [
+            {
+              address: veteran.email,
+              type: '1'
```

Comment:
Absolutely, and I'll hold off on merging until then. A concern I have is that the validator on that side is allowing extra elements through but silently dropping them, and only raising errors on malformed elements that it has a schema for.